apps/mull-ui/src/app/pages/login/login.scss

```diff
@@ -4,97 +4,101 @@
   display: grid;
   grid-template-columns: 1fr;
   grid-row-gap: 1.5rem;
-}
 
-.login-mull-logo {
-  padding-top: 4rem;
-  display: block;
-  margin: 0 auto;
-  width: 30%;
-  padding-bottom: 3rem;
-}
+  .login-mull-logo {
+    padding-top: 4rem;
+    display: block;
+    margin: 0 auto;
+    width: 30%;
+    padding-bottom: 3rem;
+  }
 
-.facebook {
-  margin-bottom: 2rem;
-  background-color: $facebook;
-  padding-left: 1.2rem;
-}
+  .facebook {
+    margin-bottom: 2rem;
```

Comment:
Not sure why this button specifically needs a margin bottom. Seem unnecessary if so please remove.
```suggestion
```